chapters/projectManagement.typ

```diff
@@ -191,24 +191,34 @@ There is almost no deviation from the plan, except for the first refactoring, wh
 To monitor the working hours effectively, a Google Sheet was established where information about the time spent was meticulously recorded.
 Each record contains a date, the amount of time spent, name of executor, task category and a brief comment detailing the specific work performed during that time.
 
-// COR Falls ihr es noch aufschlüsseln könnt, wäre eine Aufteilung auf Personen noch üblich.
 The summary in @time-tracking-report shows the total time spent on each category per week.
-Most of the time spent was invested into the documentation and implementation.
-@time-spent-per-category shows the time spent per category in percentage.
+@time-tracking-report-diagram shows the share of each category per week.
+Most of the time spent was invested into the documentation and implementation, with the former being the main focus at the end of the project.
+
+@time-spent-per-person-and-category show the time spent by each project author per category.
+Both authors contributed a similar amount to both documentation and implementation.
 
-// COR Allenfalls noch eine Diagramm für die Wochen.
-// TODO: update image
 #figure(
   image("../images/time_tracking_report.png"),
   kind: table,
   caption: [
-    Hours worked per week and category
+    Time spent per week and category
   ],
 ) <time-tracking-report>
 
 #figure(
-  image("../images/time_spent_per_category.png", width: 90%),
```

Comment:
chasch das diagramm dine lah, ich hett ez eifach na es zuesätzlichs inetah womer gseht wer wiviel gschaffet hett ned pro kategorie. Es eifachs chueche diagramm wie ih dere biispil BA womer becho hend. 

Un es bispil drin zha wär nice susch isch schwierig zum nahvollzieh wies usgseht und ebs in text guet drii passt.

Reply:
![image](https://github.com/sa-concept-refactoring/doc/assets/6904387/efbd53b0-c46b-4d2a-9b07-322d7cf1cba1)

Reply:
I adjusted diagrams in the google sheets:
- Time Invested per Category
- Time Invested per Person
- Time Invested per Category and Project Week